scripts/quests/otherAreas/A_Bitter_Past.lua

```diff
@@ -0,0 +1,162 @@
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        return quest:progressEvent(152)
+                    end
+                end,
+            },
+            ['Equette'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(153)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [152] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 1)
+                end,
+
+                [153] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [154] = function(player, csid, option, npc)
+                    if quest:complete(player) then
+                        player:delKeyItem(xi.ki.TINY_WRISTLET)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.LUFAISE_MEADOWS] =
+        {
+            ['qm_bitter_past'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 2 and
+                        quest:getVar(player, 'nmKilled') ~= 1 and
+                        npcUtil.popFromQM(player, npc, { orcNM, orcNM + 1 }, { claim = true, hide = 0 })
+                    then
+                        return quest:messageText(lufaiseID.text.SENSE_OF_FOREBODING)
+                    elseif
+                        quest:getVar(player, 'nmKilled') == 1 and
+                        not player:hasKeyItem(xi.ki.TINY_WRISTLET)
+                    then
+                        npcUtil.giveKeyItem(player, xi.ki.TINY_WRISTLET)
+                        return quest:noAction()
+                    end
+                end,
+            },
+
+            ['Blackbone_Frazdiz'] =
+            {
+                onMobDeath = function(mob, player)
+                    if
+                        quest:getVar(player, 'Prog') == 2 and
+                        GetMobByID(orcNM + 1):isDead()
+                    then
+                        quest:setVar(player, 'nmKilled', 1)
+                    end
+                end,
+            },
+
+            ['Rainbringer_Yjatvot'] =
+            {
+                onMobDeath = function(mob, player)
+                    if
+                        quest:getVar(player, 'Prog') == 2 and
+                        GetMobByID(orcNM):isDead()
+                    then
+                        quest:setVar(player, 'nmKilled', 1)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStaus.QUEST_COMPLETED and
+                quest:getVar(player, 'Prog') == 3
+        end,
```

Comment:
quest:complete deletes quest vars iirc. This will never be 3

Reply:
Yea - let me double check on this